internal/cmd/build-uki.go

```diff
@@ -878,8 +878,8 @@ func createISO(e *elemental.Elemental, sourceDir, outputDir, overlayISO, keysDir
 		return err
 	}
 
-	// Create just the size we need + 50MB just in case
-	imgSize := artifactSize + 50
+	// Create just the size we need + 1MB for safety margin
```

Comment:
is this intended?

Reply:
yes, I still not want to cut it that close in case there is some dirs around or something. we always leave some small size around just in case there is some extra configs or something similar